.github/workflows/ci.yml

```diff
@@ -468,7 +468,12 @@ jobs:
         fi
         cd ../../
         mkdir ./cppyy-backend/llvm-project/
-        cp -r ./llvm-project/ ./cppyy-backend/llvm-project/
+        OS=$(uname -s)
+        if [[ "$OS" == "Darwin" ]]; then
+          cp -r -v ./llvm-project/ ./cppyy-backend/llvm-project/
```

Comment:
Why do we have these set up differently on Darwin?

Reply:
I have no idea to be honest. On my local machine I can use the Linux version on osx, but on the Github runner the recursive copy gives different results on osx to Linux.

Reply:
On osx on the Github runner it copies recursively the files from the folder your copying from into the folder you suggest, while on Linux it copies into a folder with the same name as the one your copying (like on my local machine)

Reply:
I think we probably need to delete the backslash or something equally trivial. Would it be worth understanding and fixing it?

Reply:
I think the best solution for now is to put a fix me next to it, and fix it after https://github.com/compiler-research/cppyy-backend/pull/81 has been merged. I think I can avoid the copy command completely by defining the absolute path of what I want to cache.

Reply:
I'd define the absolute path, by first defining an environment variable equal to the top level build directory path and using that. Having this environment variable would allow me to define the working directory for each part of the job too. This should make the ci more portable to other repos in the organisation.

Reply:
@vgvassilev I've added a fixme with a comment on my suggested solution. I see that the daily run of the ci on the master branch has started. If you choose to merge this PR, remember to cancel this job if still running, and clear the cache before doing so.